src/main/java/JavaReferenceDataTypes/DescribeOutput.java

```diff
@@ -0,0 +1,32 @@
+package JavaReferenceDataTypes;
+
+public class DescribeOutput {
+    public static int[] make(int n) {
+        int[] a = new int[n];
+        for (int i = 0; i < n; i++) {
+            a[i] = i + 1; // completeaza array-ul(cu make.length = 5) cu valori, atribuind ca valoare = indexul + 1
+        }
+        return a;
+    }
+
+    public static void dub(int[] jub) {
+        for (int i = 0; i < jub.length; i++) {
+            jub[i] *= 2; //inmulteste valoarea de pe fiecare index cu 2
+        }
+    }
+
+    public static int mus(int[] zoo) {
```

Comment:
this method name - for me - is very ambiguous, I cant use it without reading it's content / functionality - and this is not a good practice.

Reply:
Done